homeassistant/components/google_assistant/trait.py

```diff
@@ -395,15 +402,6 @@ class TemperatureSettingTrait(_Trait):
         COMMAND_THERMOSTAT_TEMPERATURE_SET_RANGE,
         COMMAND_THERMOSTAT_SET_MODE,
     ]
-    # We do not support "on" as we are unable to know how to restore
-    # the last mode.
-    hass_to_google = {
```

Comment:
Please keep all mapping logic inside this trait as this is the only place where it will be used.